### PR TITLE
Update to build script, tag version

### PR DESCRIPTION
**Pull request by bombcar**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+# Do you need to test how your custom blocks interacts with a player that is not the owner? -> leave name empty
+developmentEnvironmentUserName = Developer
+
+# Define a source file of your project with:
+# public static final String VERSION = "GRADLETOKEN_VERSION";
+# The string's content will be replaced with your mod's version when compiled. You should use this to specify your mod's
+# version in @Mod([...], version = VERSION, [...])
+# Leave these properties empty to skip individual token replacements
+replaceGradleTokenInFile = Constants.java
+gradleTokenModId = GRADLETOKEN_MODID
```
Comment (SinTh0r4s) [inline gradle.properties:30]: I found issues with replacing modName and modId for dev builds. It might be better to leave them as constants for now.

File: gradle.properties [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+# that is annotated with @Mod) you want this to be true. When in doubt: leave it on false!
+containsMixinsAndOrCoreModOnly = false
+
+# If enabled, you may use 'shadowCompile' for dependencies. They will be integrated in your jar. It is your
+# responsibility check the licence and request permission for distribution, if required.
+usesShadowedDependencies = false
+
+# Optional parameter to customize the produced artifacts. Use this to preserver artifact naming when migrating older
+# projects. New projects should not use this parameter.
+#customArchiveBaseName =
```
Comment (SinTh0r4s) [inline gradle.properties:64]: Original file name is lower case. It should be kept for compatibility: `backpack`
Look for archiveBaseName in the original script or a old release jar in the pack.

**Review by SinTh0r4s — CHANGES_REQUESTED** [review]
.github is copied from ExampleMod. The mod releases a migration_pack.zip in releases. Please use those files instead! The files are not different, but some should not be added to every mod. We have not yet split the ExampleMod and the build script into two different repositories. 

You're getting the hang of it. Good job!

**Comment by bombcar** [comment]
This builds and runs butt something is wrong - it's not loading as expected in the full pack ....

File: gradle.properties [hunk 3]
```diff
@@ -0,0 +1,64 @@
+modName = Backpack
```
Comment (SinTh0r4s) [inline gradle.properties:1]: You need to keep the name from Constants.gradle! Don't change the names because you think it sounds better.

**Review by SinTh0r4s — CHANGES_REQUESTED** [review]

**Comment by bombcar** [comment]
This is weird - without fingerprint it refused to load.

Perhaps we should consider updating ExampleMod to *use* the certificate/fingerprinting stuff and make it work? Otherwise we can yeet it when it causes problems.

**Comment by SinTh0r4s** [comment]
Removing the certificate is the right step for now. Not sure why we would add certificates to the mods. It might produce some benefits, but also adds another dimension to maintain on the build scripts. I won't be bored enough to do that ever 😉

**Review by SinTh0r4s — APPROVED** [review]

**Comment by Glease** [comment]
jar signing allows modder to rule out problems exists only in fork but not in mainline. doesn't make much sense since *we are the fork*.

**Comment by bombcar** [comment]
Yeah our version numbers are our signature heheh
